### PR TITLE
Hardcoded servlet messages

### PR DESCRIPTION
**Pull request by tgomezzzz**
Another step in the week 3 tutorial, this PR adds three hardcoded messages to the Comments tab of my portfolio.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -24,9 +26,29 @@
...
+    String json = convertToJsonUsingGson(messages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(List<String> messages) {
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+    System.out.println(json);
```
Comment (vidhyal) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Was this for debugging? If so, please remove.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,156 +5,196 @@
...
+        <button class="tab-button" onclick="displayTab(event, 'Bucket List')">My Bucket List</button>
+        <button class="tab-button" onclick="displayTab(event, 'Gallery')">Gallery</button>
+        <button class="tab-button" onclick="displayTab(event, 'Comments')">Comments</button>
       </div>
-    
-      <div class="timeline-branch" id="tornadoes">
-      <img src="/images/tornado.jpg" alt="Tornadoes" style="left: 30%; top: 5%;" onclick="displayMoreInfo(event, 'tornadoesinfo')">
+      
+      <div id="About Me" class="tab-info">
+        <h3 id="about-me-header">About Me
```
Comment (vidhyal) [inline portfolio/src/main/webapp/index.html:29]: Is the entire thing a heading?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -5,156 +5,196 @@
...
+          <a href="http://www.linkedin.com/in/tomgom"><img src="images/linkedin-logo.png" style="border-radius: 10%;"/></a>
+          <a href="http://www.github.com/tgomezzzz"><img src="images/github-logo.jpg" style="border-radius: 50%;"/></a>
+        </h3>
+        <p><img src="/images/beta-headshot.jpg" align="right" id="headshot" onclick="easterEgg()"/>
+            My full name is Thomas Joaquin Kaufman-Gomez, but I usually just go by Tommy Gomez. I'm a rising junior at Columbia University
+            in New York City, and I'm studying computer science. Born and raised in Sunnyvale, California, I've lived on both coasts and
+            can confidently confirm that the West Coast is superior, although the East Coast is not to be undervalued (in fact, I 
+            would argue that both coasts are equal if it weren't for the East's weather).
+        </p>
```
Comment (vidhyal) [inline portfolio/src/main/webapp/index.html]: Please be consistent in indentation. Here you have a new line for </p> tag, while in the next instance, you add it at the end of the paragraph.

If it is a single line paragraph, it makes sense to have the closing tag at the end.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//   https://www.apache.org/licenses/LICENSE-2.0
```
Comment (vidhyal) [inline portfolio/src/main/webapp/script.js:7]: Why the extra whitespace?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -24,9 +26,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
+  @Override
+  public void init() {
+      messages = new ArrayList<>();
```
Comment (vidhyal) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Please review indentation.

**Review by vidhyal — COMMENTED** [review]
I don't think the changes made in other PRs are reflected here. Please verify.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (tgomezzzz) [reply portfolio/src/main/webapp/index.html:29]: I grouped the two images in with the header to get them to display side-by-side with the text.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (tgomezzzz) [reply portfolio/src/main/webapp/script.js:7]: This license was already included in the file wen I cloned it. Should I delete the extra whitespace?

Comment (vidhyal) [reply portfolio/src/main/webapp/script.js:7]: No, that is fine.

**Comment by tgomezzzz** [comment]
Abandoned, but not forgotten

**Pull request closed without merging**